readme_renderer/__about__.py

```diff
@@ -23,7 +23,7 @@
                'descriptions for Warehouse')
 __uri__ = "https://github.com/pypa/readme_renderer"
 
-__version__ = "18.1"
+__version__ = "19.0"
 
 __author__ = "Donald Stufft"
 __email__ = "donald@stufft.io"
```

Comment:
We have a better email address we have started using for at least some of this stuff -- see https://github.com/pypa/warehouse/blob/master/warehouse/__about__.py .

Reply:
Great, done.